working/333 - shared memory multithreading/proposal.md

```diff
@@ -842,9 +594,7 @@ class Isolate {
   /// Shared isolate contains a copy of the
   /// global `shared` state of the current isolate and does not have any
   /// non-`shared` state of its own. An attempt to access non-`shared` static variable throws [IsolationError].
-  ///
-  /// If [task] is not [Shareable] then [ArgumentError] is thrown.
-  static Future<S> runShared<S extends Shared>(S Function() task);
+  static Future<S> runShared<S>(S Function() task);
```

Comment:
I think I would prefer to have a single shared isolate - it can also become sorta the representation of the _isolate group_, for which we currently don't have any representation.